compiler/code-gen/files/init-scripts.cpp

```diff
@@ -122,8 +123,8 @@ struct RunInterruptedFunction {
 
     std::string script_start = "co_await InstanceState::get().run_instance_prologue<" + image_kind + ">();";
     std::string script_finish = "co_await InstanceState::get().run_instance_epilogue();";
-    FunctionSignatureGenerator(W) << "task_t<void> " << FunctionName(function) << "$run() " << BEGIN << script_start << NL << await_prefix
-                                  << FunctionName(function) << "();" << NL << script_finish << NL << "co_return;" << END;
+    FunctionSignatureGenerator(W) << "task_t<void> " << FunctionName(function) << "$run() " << BEGIN << script_start << NL << try_wrapper << await_prefix
+                                  << FunctionName(function) << "());" << NL << script_finish << NL << "co_return;" << NL << END;
```

Comment:
As I see we wrap every call of interrupted function. Why don't we check that this function can throw an exception? Won't overhead in this case be too much?